server/bcoinSocket.js

```diff
@@ -5,11 +5,11 @@ const socketHandler = (nodeClient, walletClient) => {
   nodeClient.on('error', err => logger.error('Socket error (node): ', err));
   walletClient.on('error', err => logger.error('Socket error (wallet): ', err));
   logger.info('Set up socket error handlers');
-
   const walletPrefix = 'wallet ';
   const subscriptions = {}; // cache to manage subscriptions made by clients
 
   return async socket => {
+    logger.info('New socket connection. Attaching handlers');
```

Comment:
Is there any way to identify which socket is new in this logline?

Reply:
Yes! Kind of. We can get the IP and Port of the socket. This is actually getting added in in the socket manager during the auth stage. This was mainly added now for testing, but it will actually be superseded by the manager.

Reply:
Added!